packages/background/package.json

```diff
@@ -1,28 +1,30 @@
 {
   "name": "helios-background",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "type": "module",
   "main": "src/index.js",
+  "private": true,
   "author": "yqrashawn <namy.19@gmail.com>",
   "license": "MIT",
   "dependencies": {
     "@cfxjs/db": "workspace:packages/db",
+    "@cfxjs/extension-runtime": "workspace:packages/extension-runtime",
+    "@cfxjs/fluent-wallet-consts": "workspace:packages/consts",
+    "@cfxjs/fluent-wallet-inner-utils": "workspace:packages/inner-utils",
     "@cfxjs/rpc-engine": "workspace:packages/rpc-engine",
     "@cfxjs/transducers": "workspace:packages/transducers",
-    "@cfxjs/wallet_addVault": "workspace:packages/rpcs/wallet_addVault",
-    "@cfxjs/wallet_generateMnemonic": "workspace:packages/rpcs/wallet_generateMnemonic",
-    "@cfxjs/wallet_generatePrivateKey": "workspace:packages/rpcs/wallet_generatePrivateKey",
-    "@cfxjs/wallet_getVaults": "workspace:packages/rpcs/wallet_getVaults",
-    "@cfxjs/wallet_importAddress": "workspace:packages/rpcs/wallet_importAddress",
-    "@cfxjs/wallet_importPrivateKey": "workspace:packages/rpcs/wallet_importPrivateKey",
+    "@cfxjs/wallet_add-vault": "workspace:packages/rpcs/wallet_addVault",
+    "@cfxjs/wallet_generate-mnemonic": "workspace:packages/rpcs/wallet_generateMnemonic",
+    "@cfxjs/wallet_generate-private-key": "workspace:packages/rpcs/wallet_generatePrivateKey",
+    "@cfxjs/wallet_get-vaults": "workspace:packages/rpcs/wallet_getVaults",
+    "@cfxjs/wallet_import-address": "workspace:packages/rpcs/wallet_importAddress",
+    "@cfxjs/wallet_import-private-key": "workspace:packages/rpcs/wallet_importPrivateKey",
```

Comment:
The name of the npm packages has mixed use of `-` and `_`. Is it better to use `-` totally, and the workspace package name could follow the same baseline~